sql/1.095/github_user.sql

```diff
@@ -0,0 +1,8 @@
+BEGIN;
+    CREATE TABLE github_user_link (
+        "users_id" bigint NOT NULL,
+        "data" text NOT NULL
+    );
+
+    ALTER TABLE users ADD COLUMN "github_user" text;
+COMMIT;
```

Comment:
Just noticing the users `ALTER` - we need to remove the CREATE TABLE block or this will fail.